### PR TITLE
Moved note about Global Ajax events to notes.xsl

### LINKED ISSUE #798: Move sentence about Global Ajax events as a global note
In all the methods related to [jQuery global Ajax events](http://api.jquery.com/category/ajax/global-ajax-event-handlers/), there's a note like:

> As of jQuery 1.8, the [NAME OF THE METHOD] method should only be attached to document.

I think we can move this as a global note (inside the file `notes.xsl`) so that it'll become easier to manage and to update.

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
Fixes #798

### CODE REVIEW

File: notes.xsl [hunk 1]
```diff
@@ -68,7 +68,7 @@
 			As the <code>.<xsl:value-of select="@data-event"/>()</code> method is just a shorthand for <code>.on( "<xsl:value-of select="@data-event"/>", handler )</code>, detaching is possible using <code>.off( "<xsl:value-of select="@data-event"/>" )</code>.
 		</xsl:when>
 		<xsl:when test="@id = 'global-ajax-event'">
-			As of jQuery 1.9, all the handlers for the <a href="/category/ajax/global-ajax-event-handlers/">jQuery global Ajax events</a>, including those added with the <code><xsl:value-of select="@data-title"/></code> method, <em>must</em> be attached to <code>document</code>.
+         As of jQuery 1.8, the <code><xsl:value-of select="@data-title"/></code> method <em>should</em> only be used to attach handlers to <code>document</code>. As of jQuery 1.9, all the handlers for the <a href="/category/ajax/global-ajax-event-handlers/">jQuery global Ajax events</a>, including those added with the <code><xsl:value-of select="@data-title"/></code> method, <em>must</em> be attached to <code>document</code>.
```
Comment (arthurvr) [inline notes.xsl]: Can you fix the indentation here?

**Comment by AurelioDeRosa** [comment]
Final review for this PR?

**Comment by dmethvin** [comment]
Looks accurate to me. I'm not sure we need this text in the note though. That text was important in the era between 1.8 and 1.9 to warn people it was going away, but now that it has gone away it seems that the only really important milestone from a docs standpoint is 1.9 where the behavior changed.

**Comment by AurelioDeRosa** [comment]
So, should I keep the deletion of the note in all the files and keep in the `notes.xml` file only the sentence about 1.9?

**Comment by dmethvin** [comment]
@AurelioDeRosa that sounds like a plan!

**Comment by AurelioDeRosa** [comment]
PR updated. I'll wait for a final LGTM before merging. Thank you @dmethvin

**Comment by dmethvin** [comment]
LGTM thanks!

**Pull request closed without merging**